frontend/src/components/sagas/SagaDashboard.vue

```diff
@@ -0,0 +1,603 @@
+                { value: 'UPDATE_DIGITAL_ID', text: 'UPDATE DIGITAL ID'},
+                { value: 'GET_PEN_REQUEST', text: 'GET PEN REQUEST'},
+                { value: 'UPDATE_PEN_REQUEST', text: 'UPDATE PEN REQUEST'}
+              ]
+            }
+          ]
+        },
+        {
+          value: 'replication',
+          text: 'Replication',
+          sagaNames: [
+            {
+              value: 'PEN_REPLICATION_STUDENT_UPDATE_SAGA',
+              text: 'PEN REPLICATION STUDENT UPDATE',
+              states: [
+                { value: 'INITIATED', text: 'INITIATED'},
+                { value: 'GET_STUDENT', text: 'GET STUDENT'},
+                { value: 'CREATE_STUDENT', text: 'CREATE STUDENT'},
+                { value: 'UPDATE_STUDENT', text: 'UPDATE STUDENT'},
+                { value: 'UPDATE_PEN_DEMOG', text: 'UPDATE PEN DEMOG'},
+                { value: 'UPDATE_PEN_DEMOG_TRANSACTION', text: 'UPDATE PEN DEMOG TRANSACTION'}
+              ]
+            },
+            {
+              value: 'PEN_REPLICATION_STUDENT_CREATE_SAGA',
+              text: 'PEN REPLICATION STUDENT CREATE',
+              states: [
+                { value: 'INITIATED', text: 'INITIATED'},
+                { value: 'GET_NEXT_PEN_NUMBER', text: 'GET NEXT PEN NUMBER'},
+                { value: 'CREATE_STUDENT', text: 'CREATE STUDENT'},
+                { value: 'ADD_PEN_DEMOG', text: 'ADD PEN DEMOG'},
+                { value: 'UPDATE_PEN_DEMOG_TRANSACTION', text: 'UPDATE PEN DEMOG TRANSACTION'}
+              ]
+            },
+            {
+              value: 'PEN_REPLICATION_POSSIBLE_MATCH_CREATE_SAGA',
+              text: 'PEN REPLICATION POSSIBLE MATCH CREATE',
+              states: [
+                { value: 'INITIATED', text: 'INITIATED'},
+                { value: 'ADD_POSSIBLE_MATCH', text: 'ADD POSSIBLE MATCH'},
+                { value: 'CREATE_PEN_TWINS', text: 'CREATE PEN TWINS'},
+                { value: 'UPDATE_PEN_TWIN_TRANSACTION', text: 'UPDATE PEN TWIN TRANSACTION'}
+              ]
+            },
+            {
+              value: 'PEN_REPLICATION_POSSIBLE_MATCH_DELETE_SAGA',
+              text: 'PEN REPLICATION POSSIBLE MATCH DELETE',
+              states: [
+                { value: 'INITIATED', text: 'INITIATED'},
+                { value: 'DELETE_POSSIBLE_MATCH', text: 'DELETE POSSIBLE MATCH'},
+                { value: 'DELETE_PEN_TWINS', text: 'DELETE PEN TWINS'},
+                { value: 'UPDATE_PEN_TWIN_TRANSACTION', text: 'UPDATE PEN TWIN TRANSACTION'}
+              ]
+            }
+          ]
+        }
+      ],
+      sagaStatuses: [
+        { value: 'STARTED', text: 'STARTED'},
+        { value: 'IN_PROGRESS', text: 'IN PROGRESS'},
+        { value: 'COMPLETED', text: 'COMPLETED'},
+        { value: 'FORCE_STOPPED', text: 'FORCE STOPPED'}
+      ],
+      searchCriteria: {'createDate': {}, 'updateDate': {}},
+      searchLoading: false,
+      statsLoading: {
```

Comment:
How about moving `statsLoading` definition to every saga type in `sagaTypes`? so only `sagaTypes` need to be changed if new sagaType is added.